### PR TITLE
Bug 1986305: drop operand images as required field on CRD

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

File: config/crd/bases/nfd.openshift.io_nodefeaturediscoveries.yaml [hunk 1]
```diff
@@ -35,36 +34,72 @@ spec:
           metadata:
             type: object
           spec:
-            description: NodeFeatureDiscoverySpec defines the desired state of NodeFeatureDiscovery
+            description: Specification of the desired behavior of the Node Feature
+              Discovery
```
Comment (wabouhamad) [inline config/crd/bases/nfd.openshift.io_nodefeaturediscoveries.yaml:38]: Is "Discovery" on line 38 supposed to be on previous line 37 ?

**Review by wabouhamad — COMMENTED** [review]
some lines in description are spanning to the next line

File: config/crd/bases/nfd.openshift.io_nodefeaturediscoveries.yaml (hunk 1 above)
Comment (ArangoGutierrez) [reply config/crd/bases/nfd.openshift.io_nodefeaturediscoveries.yaml:38]: This is automated by `make manifests` to fit the best practice of 80 column text

**Pull request merged**